### DISCUSSION TITLE
6.0.0-beta.7

### DISCUSSION THREAD

**Discussion by arturcic** [GitTools/GitVersion#3962] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/GitTools/GitVersion/releases/tag/6.0.0-beta.7'>6.0.0-beta.7</a>.</em>

**Comment by alipievemil** [comment]
prevent-increment-of-merged-branch-version not found? what was changed?

**Reply by arturcic** [reply]
This has changed to this syntax:

```yaml
    prevent-increment:
      of-merged-branch: true
```
